src/app/styles/base/_variables.scss

```diff
@@ -0,0 +1 @@
+$error-message-color: #ED4337;
```

Comment:
The src/_variables wasn't supposed to be there, I have deleted it now. 
_variable in base folder is for base files which are valid throughout the app. 
_variable in components folder is for the component files valid for a specific components.